### PR TITLE
CCloudify convert transform stream

### PR DESCRIPTION
**Pull request by awalther28**
### Description

CCloudify transform stream pattern. Verified confluent walkthrough locally. Validated semaphore test [passes](https://confluentinc.semaphoreci.com/jobs/84005d4c-2db7-42ee-b05e-31001a395497). Failing tests are unrelated.

Added a short answer to confluent and kstreams

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/convert-transform-stream/transform-a-stream-of-events/confluent.html

### New tutorial checklist

- [x] Add a `Short Answer`

### CODE REVIEW

File: _includes/tutorials/transforming/kstreams/markup/answer/short-answer.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Use the `map()` method to take each input record and create a new stream with transformed records in it. The example below uses a custom function `convertRawMovie()` to perform the transformation logic.
```
Comment (ybyzek) [inline _includes/tutorials/transforming/kstreams/markup/answer/short-answer.adoc:1]: @awalther28 "perform the transformation logic" may sound a bit vague to users.  I defer to @bbejeck and you for more accurate wording, but maybe rephrase to something along the lines of "convert the object type"

File: _includes/tutorials/transforming/confluent/markup/dev/run-producer.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+In a new terminal, run:
+
+```
+ccloud kafka topic produce raw-movies \
```
Comment (ybyzek) [inline _includes/tutorials/transforming/confluent/markup/dev/run-producer.adoc:4]: In the staging site, the following lines are coming out formatted strangely.  Maybe convert this to a `.sh` file and include that file...like shown above (that would also follow other adoc patterns)

File: _includes/tutorials/transforming/confluent/markup/dev/run-consumer.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Leave your original terminal running. To consume the events produced by your Streams application you'll need another terminal open.
+
+First, to consume the events of drama films, run the following:
+
+```
+ccloud kafka topic consume movies \
```
Comment (ybyzek) [inline _includes/tutorials/transforming/confluent/markup/dev/run-consumer.adoc:6]: Same here, move command to `.sh` file?

**Review by ybyzek — CHANGES_REQUESTED** [review]
@awalther28 thanks for the PR.  Left some comments.

File: _includes/tutorials/transforming/confluent/markup/dev/run-producer.adoc (hunk 2 above)
Comment (awalther28) [reply _includes/tutorials/transforming/confluent/markup/dev/run-producer.adoc]: I've broken this out into a separate `.sh` file in commit https://github.com/confluentinc/kafka-tutorials/pull/848/commits/34c69403818b635bab2b219cda65a66043aef8b3 and that has resolved the strange rendering (verified locally) :)

File: _includes/tutorials/transforming/confluent/markup/dev/run-consumer.adoc (hunk 3 above)
Comment (awalther28) [reply _includes/tutorials/transforming/confluent/markup/dev/run-consumer.adoc]: Same as above

File: _includes/tutorials/transforming/kstreams/markup/answer/short-answer.adoc (hunk 1 above)
Comment (awalther28) [reply _includes/tutorials/transforming/kstreams/markup/answer/short-answer.adoc]: I reworded this to --> "Use the `map()` method to take each input record and create a new stream with transformed records in it. The records are transformed via a custom function, in this case `convertRawMovie()`." 

But maybe that's still a little too vague 🤷‍♀️

**Review by ybyzek — APPROVED** [review]
LGTM thanks @awalther28
